.github/workflows/bff-ci.yml

```diff
@@ -47,13 +47,12 @@ jobs:
       run: dotnet build bff.sln -c Release
     - name: Dotnet devcerts
       run: dotnet dev-certs https --trust
+    - name: Install Playwright
+      run: pwsh test/Hosts.Tests/bin/Release/net9.0/playwright.ps1 install --with-deps
     - name: Test
       run: dotnet test bff.sln -c Release --no-build --logger "console;verbosity=normal" --logger "trx;LogFileName=Tests.trx" --collect:"XPlat Code Coverage"
```

Comment:
Because Playwright is slow (slow to install because it downloads browsers and slow to run), to keep CI build times sane, I predict you'll want to run your playwright tests in a separate parallel job.

Reply:
I'll keep an eye out for it. For now, 3 minutes of buildtime is something can live with. 

The issue with carving out the playwright tests is that that path (testing including playwright) will take 3 minutes, so overall build time is not improved. To run the playwright tests, I have to build everything, run everything, then execute the playwright tests (which are quite fast).